docs/content/docs/start/quick-start.md

```diff
@@ -69,19 +69,19 @@ local in-memory service called Tinylicious, which runs on port 7070 by default.
 
 To use an Azure Fluid Relay instance instead, replace the configuration values with your Azure Fluid Relay tenant ID,
 orderer, and storage URLs that were provided as part of the onboarding process. Then pass that configuration object into
-the `FrsClient` constructor:
+the `AzureClient` constructor:
 
 ```typescript
-// This configures the FrsClient to use a remote Azure Fluid Service instance.
-const config: FrsConnectionConfig = {
+// This configures the AzureClient to use a remote Azure Fluid Service instance.
+const config: AzureConnectionConfig = {
     tenantId: "myFrsTenantId",
```

Comment:
same comment on naming here.  should these also be changed to "Azure"?  is "FRS" still a valid abbreviation in any circumstances?

Reply:
Dropped the "frs" prefix on them 😄 And I believe the term "FRS" itself is still used to refer to the service but it isn't the "official" name any longer whenever we refer to it in code

Reply:
Also, for the test scripts, is it okay if I update "test:realsvc:frs" to  "test:realsvc:azure" or will that break a build?